action.yml

```diff
@@ -47,7 +47,7 @@ runs:
         set -x
         mkdir -p test-data
         cp -r ${{ inputs.path }} test-data
-        python3 -uS ${{ github.action_path }}/bin/generate-metadata.py > test-data/metadata.json
+        python3 -uS ${{ github.action_path }}/bin/generate-metadata > test-data/metadata.json
```

Comment:
Why remove the .py?

Reply:
The language is (or, should be) irrelevant to running a script.
It's just convention, shrug.